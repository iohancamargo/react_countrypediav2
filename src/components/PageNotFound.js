import React from 'react';
import { Link } from 'react-router-dom';
import { withRouter } from "react-router";

const PageNotFound = () => {
    return (
        <div className="not-found parallax">
            <div className="sky-bg"></div>
            <div className="wave-7"></div>
            <div className="wave-6"></div>

            <Link to="/" className="wave-island">
                <img src="/images/island.svg" alt="Island" />
            </Link>
            <div className="wave-5"></div>
            <div className="wave-lost wrp">
                <span>4</span>
                <span>0</span>
                <span>4</span>
            </div>

            <div className="wave-4"></div>
            <div className="wave-boat">
                <img className="boat" src="/images/boat.svg" alt="Boat" />
            </div>
            <div className="wave-3"></div>
            <div className="wave-2"></div>
            <div className="wave-1"></div>
            <div className="wave-message">
                <p>Your're lost</p>
                <p>Click on the island to return</p>
            </div>
        </div>
    );
};

export default withRouter(PageNotFound);
