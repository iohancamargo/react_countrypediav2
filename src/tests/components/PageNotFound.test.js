import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import PageNotFound from '../../components/PageNotFound';

configure({adapter: new Adapter()});

test('should render PageNotFound correctly', () => {
    const wrapper = shallow(<PageNotFound />);
	expect(wrapper).toMatchSnapshot();
});