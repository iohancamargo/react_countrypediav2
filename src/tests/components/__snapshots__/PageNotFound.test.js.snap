// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageNotFound correctly 1`] = `ShallowWrapper {}`;
